clients/privacy-center/cypress/support/commands.ts

```diff
@@ -60,6 +60,18 @@ Cypress.Commands.add("visitConsentDemo", (options?: FidesConfig) => {
     onBeforeLoad: (win) => {
       // eslint-disable-next-line no-param-reassign
       win.fidesConfig = options;
+
+      // Add event listeners for Fides.js events
+      win.addEventListener(
+        "FidesInitialized",
+        cy.stub().as("FidesInitialized")
+      );
+      win.addEventListener("FidesUpdated", cy.stub().as("FidesUpdated"));
+
+      // Add GTM stub
+      // eslint-disable-next-line no-param-reassign
+      win.dataLayer = [];
+      cy.stub(win.dataLayer, "push").as("dataLayerPush");
```

Comment:
FWIW these aliases are pretty mysteriously named and non-obvious, but I couldn't come up with a more self-documenting way to include them 🙃